src/main/java/com/opentable/versionedconfig/GitProperties.java

```diff
@@ -24,34 +26,26 @@
  * Git metadata properties.
  */
 public class GitProperties {
-    private final URI remoteRepository;
-    private final String username;
-    private final String password;
+    private final List<URI> remoteRepositories;
     private final File localRepository;
```

Comment:
Is there a reason we use File here and Path in VersionedConfigUpdate?

Reply:
Probably was written before the author knew about Path.